### PR TITLE
day 4 glue check & other misc edits

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ A cheerful sound rings out, terribly out-of-place.
 [sha:17e7cd0042013da2d82d7878318adf9d7b146276]{
 -- Page 10029, Offset 7781.
 -- 先ほどの箱のとなりにある、もう一つの箱が到着する。
+// cab? is this another term for an elevator?
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:19]: i think this was one we discussed a long time ago. cab is the term for the actual elevator doo-hicky while "elevator" is the whole system. I'm fine changing this to elevator

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 2]
```diff
@@ -365,7 +366,7 @@ They're burning. The flames are definitely flickering.
 [sha:26ef150467e16696288b328e28d864bc36843b62]{
 -- Page 829, Offset 7850.
 -- <な|・><の|・><に|・><建|・><物|・><は|・><燃|・><え|・><て|・><い|・><な|・><い|・>。
-And yet the building is not being burned.
+%{i}And yet the building is not being burned%{/i}.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:369]: and yet,

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 3]
```diff
@@ -856,7 +857,7 @@ With the blade still hidden under his coat, he tries to take a step toward Arcue
 [sha:02bc565b73d08094d15250c1beb9efc8bf02ae41]{
 -- Page 861, Offset 7948.
 -- それとも貴様の<親|・>は、最低限の<礼儀|ことば>も教えぬ形なしだったのか？”」
-Or wer't thy father and thy mother so lax in their duties that they neglected to teach thee the most fundamental of manners?"
+Or wer't thy %{i}father%{/i} and thy %{i}mother%{/i} so lax in their duties that they neglected to teach thee the most fundamental of manners?"
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:860]: italics unnecessary here.

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 4]
```diff
@@ -902,7 +903,7 @@ But, as of yet, I have no name to share with the likes of you."
 [sha:c54660cb0a9bb881fa379834bd45fdcdeeb7dd72]{
 -- Page 864, Offset 7957.
 -- 元の名前は失った、か……でも呼び名ぐらいはあるでしょう？　継いだばかりのようだけど、<仮|・><に|・><も|・><祖|・><の|・><一|・><角|・>なんだから」
-So you have lost your original name, huh... but you do have something to be called by, do you not? You may have only recently succeeded to your role, but you still number among the Ancestors."
+So you have lost your original name, huh... but you do have something to be called by, do you not? You may have only recently succeeded to your role, but you still %{i}number among the Ancestors%{/i}."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:906]: italics unnecessary

File: script/Arcueid/Day 4/04_00_ARC04_4_1.txt [hunk 5]
```diff
@@ -143,7 +143,7 @@ I'm Arcueid―――well, it's a bit of a long name, so you can just call me Arc
 [sha:635382ea5817fe1d87e546dc9a8efa4650575618]{
 -- Page 336, Offset 6483.
 -- <真祖|しんそ>って区分けされる吸血鬼だけど、あなたはなんていう人？」
-I'm from the True Ancestor caste of vampires, what sort of human are you?"
+I'm from the %{i}True Ancestor%{/i} caste of vampires, what sort of human are you?"
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_4_1.txt:146]: don't think italics are necessary here

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 6]
```diff
@@ -336,8 +336,7 @@ But you're a vampire too, aren't you?"
 [sha:fd159e9e69cc3fd11b0b1b75df619cef540f27d1]{
 -- Page 389, Offset 6640.
 -- 俺が言いたいのは、どうして<吸|・><血|・><鬼|・>であるおまえが<吸|・><血|・><鬼|・>を殺すのかって事だ。それ、おかしいだろ、理屈的に」
-// ruby dots <吸|・><血|・><鬼|・>であるおまえが<吸|・><血|・><鬼|・>
-What I'm trying to say is, why would a vampire like yourself be going out killing other vampires? That doesn't make any logical sense."
+What I'm trying to say is, why would a %{i}vampire like yourself%{/i} be going out killing %{i}other vampires%{/i}? That doesn't make any logical sense."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:339]: %{i}vampire%{/i} like yourself

File: script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 7]
```diff
@@ -391,7 +391,7 @@ are all spinning like an emergency light.
 [sha:317bb9f7f0e64fb7534ddb26669cae351ffb2720]{
 -- Page 932, Offset 8137.
 -- ごうごう。ごうごう。ごうごう。
-%{i}Rumble. Rumble. Rumble.%{/i}
+%{i}Rumble%{/i}. %{i}Rumble%{/i}. %{i}Rumble%{/i}.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_1.txt:394]: you can just put this whole line in italics

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt [hunk 8]
```diff
@@ -306,7 +306,7 @@ I never thought I would see something like this in my lifetime.
 [sha:6b0c313d7f3ac0d7fa52f04b9c1603c56c49dc01]{
 -- Page 443, Offset 6786.
 -- 「…………はあ」
-"......Hah."
+"......%{i}Sigh%{/i}."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2.txt:309]: check in game but i don't think this one is actually a sigh. More like an exasperated exhale. Might need to go back to Hah but you can put it in italics.

File: script/Arcueid/Day 4/04_00_ARC04_6_2B.txt [hunk 9]
```diff
@@ -432,8 +432,7 @@ Instead, they use humans who are frail but have fewer weaknesses, to gather bloo
 [sha:8d0f70688bacb5a638a76c9819eec7229aaf061b]{
 -- Page 502, Offset 6945.
 -- <敵|・><が|・><多|・><い|・>、という事かしら」
-// ruby dots <敵|・><が|・><多|・><い|・>
-They have a lot of enemies, or something like that."
+They have %{i}a lot of enemies%{/i}, or something like that."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:435]: only "a lot" in italics

File: script/Ciel/Day 4/04_01_CIEL04_4_2.txt [hunk 10]
```diff
@@ -26,7 +26,7 @@ She's got me there. On a scale of normal to abnormal, my eyes are in the same ca
 [sha:0dbd051ed759abe6d08ef992699e2f9e0491b258]{
 -- Page 96, Offset 8415.
 -- ……けど、それだって<こ|・><こ|・><ま|・><で|・>じゃなかった。
-...But even my eyes aren't at this level.
+...But even %{i}my eyes%{/i} aren't at this level.
```
Comment (Avuxo) [inline script/Ciel/Day 4/04_01_CIEL04_4_2.txt:29]: just "my" in italics

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 4/04_00_ARC04_9.txt [hunk 11]
```diff
@@ -66,7 +66,7 @@ Even humans get hungry just by simply existing. So if you vampires need to suck
 [sha:bb7a36200562d8969ee7e53dade016b6be0cb504]{
 -- Page 691, Offset 7408.
 -- 「わたしは自分から血を吸うなんて、この八百年、一度もないかな。ふつうの人間を殺した事だって、一度もないよ」
-"As for myself, I don't think I've sucked blood even once in the past eight hundred years. On top of that, I've never killed an ordinary person either."
+"As for myself, I don't think I've sucked blood even once in the past 800 years. On top of that, I've never killed an ordinary person either."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_9.txt:69]: I'd keep this written

File: script/Arcueid/Day 4/04_00_ARC04_6.txt (hunk 6 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 4/04_00_ARC04_6.txt:339]: should i keep the other italic?

File: script/Arcueid/Day 4/04_00_ARC04_6_1.txt (hunk 7 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 4/04_00_ARC04_6_1.txt:394]: whoops my bad

File: script/Ciel/Day 4/04_01_CIEL04_4_2.txt (hunk 10 above)
Comment (rschlaikjer) [reply script/Ciel/Day 4/04_01_CIEL04_4_2.txt:29]: Shouldn't the italics be on _this_ here?

File: script/Arcueid/Day 4/04_00_ARC04_1.txt [hunk 12]
```diff
@@ -231,7 +231,7 @@ The time is still six in the morning.
 [sha:e8c05a7296b75eaec29bdcbf5c1a437a9e47aa59]{
 -- Page 20, Offset 5615.
 -- 何をするでもなく、20分間ぼんやりと部屋の天井を見上げ続けた。
-I stare up at the ceiling of my room for 20 minutes without doing anything.
+I stare up at the ceiling of my room for twenty minutes without doing anything.
```
Comment (IrishKnight0) [inline script/Arcueid/Day 4/04_00_ARC04_1.txt:234]: It's in number format in jp :/

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt (hunk 8 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 4/04_00_ARC04_6_2.txt:309]: i went back to it but i'm not sure what's the difference

File: script/Arcueid/Day 4/04_00_ARC04_1.txt (hunk 12 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 4/04_00_ARC04_1.txt:234]: yeah but we're not following that

File: script/Arcueid/Day 4/04_00_ARC04_6_2B.txt [hunk 13]
```diff
@@ -232,7 +232,7 @@ Humans that have their blood sucked by the vampire die and become puppets of the
 [sha:edaffa9f8bf44e96c0137a88efc16a60746acc0c]{
 -- Page 489, Offset 6905.
 -- ようは『人格の残った死体』ね。<死|し><徒|と>の<階梯|かいてい>でいうと<不死級|さんだんかいめ>の、上級の使い魔ってところ」
-In other words, a 'corpse with a personality'. Ranked against the other Dead Apostles, these undead are the third lowest tier, about equal to an Upper Mage."
+In other words, a 'corpse with a personality'. Ranked against the other Dead Apostles, these %{i}Undead%{/i} are the third lowest tier, about equal to an Upper Mage."
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:235]: isn't she talking about the rank III here? 
https://github.com/Tsukihimates/Tsukihime-Translation/blob/a87ed218660f659128ca557e556aeff3f4f6808e/script/Arcueid/Day%204/04_00_ARC04_6_2B.txt#L270

File: script/Ciel/Day 4/04_01_CIEL04_4_2.txt (hunk 10 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 4/04_01_CIEL04_4_2.txt:29]: which is it then?

File: script/Arcueid/Day 4/04_00_ARC04_6_2B.txt (hunk 13 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:235]: Oh, oops. Yeah I just scrolled up and saw 
> 「まずは初手、進行１。

You're correct

[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:235]: we put Undead in this context in caps though? should i change it?
https://github.com/Tsukihimates/Tsukihime-Translation/blob/a87ed218660f659128ca557e556aeff3f4f6808e/script/Arcueid/Day%208/08_00_ARC08_6B.txt#L137
https://github.com/Tsukihimates/Tsukihime-Translation/blob/a87ed218660f659128ca557e556aeff3f4f6808e/script/Arcueid/Day%208/08_00_ARC08_6B.txt#L216

File: script/Arcueid/Day 4/04_00_ARC04_6_2.txt (hunk 8 above)
Comment (Avuxo) [reply script/Arcueid/Day 4/04_00_ARC04_6_2.txt]: I'd say safer to stick to Haa fwiw

File: script/Ciel/Day 4/04_01_CIEL04_4_2.txt (hunk 10 above)
Comment (Avuxo) [reply script/Ciel/Day 4/04_01_CIEL04_4_2.txt:29]: yeah, for whatever reason I read this as "even my eyes [in comparison to yours]" but Ross is right.

File: script/Arcueid/Day 4/04_00_ARC04_10.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 4/04_00_ARC04_10.txt:19]: I guess if its come up this much I prefer elevator.

File: script/Arcueid/Day 4/04_00_ARC04_1.txt (hunk 12 above)
Comment (Avuxo) [reply script/Arcueid/Day 4/04_00_ARC04_1.txt:234]: I almost always prefer twenty to be written out. There are exceptions but as a general rule "it's like that in the jp" isn't a particularly solid argument.

**Pull request merged**